ext/opentelemetry-ext-opencensusexporter/src/opentelemetry/ext/opencensusexporter/metrics_exporter/__init__.py

```diff
@@ -65,6 +67,8 @@ def __init__(
             self.client = client
 
         self.node = utils.get_node(service_name, host_name)
+        self.exporter_start_timestamp = Timestamp()
```

Comment:
Is the start time for metrics really when the exporter is instantiated? I would think it would be the first time that a metric is recorded.

Reply:
I think your right, that would be more accurate. I don't see any way to get that in the SDK though? All I see is `last_update_timestamp` in the aggregator

Reply:
Yeah. It'll be kind of strange for that value to exist as part of the metric instrument. Especially for the cases when the batcher is not stateful. I'm not really sure what to do for this. Maybe bring this up to the metrics SIG?